packages/babel-types/src/index.js

```diff
@@ -240,6 +240,16 @@ export function clone(node: Object): Object {
 }
 
 /**
+ * Create a shallow clone of a `node` excluding `_private` and location properties.
```

Comment:
it says excluding `_private` but that doesn't seem to be the case. Also, if this functions is used once, consider using it inline. This is a public API and we should be careful with adding more thing to ti.